gsuite_reports_rules/gsuite_drive_visibility_change.py

```diff
@@ -1,42 +1,126 @@
 from panther_base_helpers import deep_get
 from panther_base_helpers import gsuite_parameter_lookup as param_lookup
 
+EXCLUDED_DOMAINS = {"example.com"}
+
+VISIBILITY = {"people_with_link", "public_on_the_web", "shared_externally", "unknown"}
+
+ALERT_DETAILS = {}
+
+
+def init_alert_details(log):
+    global ALERT_DETAILS  # pylint: disable=global-statement
+    ALERT_DETAILS[log] = {
+        "ACCESS_SCOPE": "<UNKNOWN_ACCESS_SCOPE>",
+        "DOC_TITLE": "<UNKNOWN_TITLE>",
+        "NEW_VISIBILITY": "<UNKNOWN_VISIBILITY>",
+        "TARGET_USER_EMAILS": {"<UNKNOWN_USER>"},
+        "TARGET_DOMAIN": "<UNKNOWN_DOMAIN>",
+    }
+
 
 def rule(event):
     if deep_get(event, "id", "applicationName") != "drive":
         return False
 
+    log = event.get("p_row_id")
```

Comment:
perhaps rename to "id" or "row_id"?

Reply:
Is a row not a "log"? I called it "log" in the `unusual_login` rule

Reply:
log works - I guess I was thinking the event is the log and the id is the id associated with it.  But I am personally terrible at naming things, so perhaps go with your gut instinct on naming this.